### DISCUSSION TITLE
Added https://www.stirlingpdf.com/ to the footer, We have paid support ...

### DISCUSSION THREAD

**Discussion by b2sc** [Stirling-Tools/Stirling-PDF#1962] [General]
The release notes say:

> Added https://www.stirlingpdf.com/ to the footer, We have paid support coming for all companies that want it, Please sign up in Contact Us form to find out more in future! Surveys to come...

Does this mean a footer is added to the PDFs? Or is it added to the application's web GUI?

**Comment by Frooodle** [comment]
Bottom of website 
We don't ever add any watermarks or anything unexpected to any pdfs

**Reply by Frooodle** [reply]
https://stirlingpdf.io/
You can see the powered by Stirling pdf now links to website
Nothing else :)

**Reply by b2sc** [reply]
Thank you so much for explaining. Sorry for being nervous. ;)
